libexec/scoop-info.ps1

```diff
@@ -1,8 +1,9 @@
-# Usage: scoop info <app>
+# Usage: scoop info <app> [options]
 # Summary: Display information about an app
-param($app)
+# Options:
+#   -v, -verbose        Show full paths and URLs
+param([string]$app, [switch]$verbose)
```

Comment:
I actually want to avoid getopt. It causes a lot of issues by locking up arguments. We're refactoring a lot of stuff anyway, so I thought we should use PS native methods.

Reply:
The advantage of `getopt` is it follows Unix param syntax, i.e., -v/--verbose, and this is what other scoop command does.

In PS's switch param type, `-verbose` (single dash) is used, so I'd preferred `getopt` at least now.

Reply:
Okay. Sure.

Unlike `scoop shim` the position/number of args does not really matter here, so I will change it to getopt.

Reply:
Actually, I just noticed. Only one command uses the `verbose` switch, which is `scoop alias`. And it uses PS params. 

https://github.com/ScoopInstaller/Scoop/blob/b95ccbe14a8c3416c1358a7a96bf2d86baa3e183/libexec/scoop-alias.ps1#L23

It was added in https://github.com/ScoopInstaller/Scoop/pull/2163

Reply:
Hmm, @r15ch13 made a wrong param 😮 

Just as our discussion under `scoop-shim`, PS's switch couldn't handle `--arg` form which is preferred in scoop cli, that's why `getopt` is used. Except for the special case in `scoop-shim` that unknown additional args should be passed to the script, `getopt` is a better choice.